packages/terra-application-header/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "terra-application-links",
+  "main": "lib/ApplicationLinks.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
```

Comment:
Update description to: The ApplicationHeader is a HOC that combines config and menu subcomponents, which are then placed within the header layout.